README.rst

```diff
@@ -78,3 +78,39 @@ Repositories
 .. _`clear-containers-image`: https://download.clearlinux.org/current/
 
 .. _`clear-containers-selinux`: https://github.com/clearcontainers/proxy/tree/master/selinux
+
+Local Building
+==============
+
+Any user can build their own Clear Containers packages using the source files provided in this
+repository, either using the OBS platform or in their own machines using the osc (Open Build Service command line tool).
+
+A Dockerfile is provided to build a container that has all the tools needed to perform local builds.
+
+.. code-block:: shell
+
+   # cd into this repository
+   cd < this repo location>
+
+   # build the container image
+   # cc-osc name is used for the tag, it can be anything.
+   docker build -t cc-osc .
+
+   # If you are behind a proxy
+   docker build --build-arg=http_proxy=http://<host>:<port> --build-arg=https_proxy=http://<host>:<port> -t cc-osc .
+
+   # Run the container and bind this repository
+   docker run -v /repo/location:/root/packaging -it cc-osc bash
+
+   # Inside the container, cd to a project, runtime in this case
+   cd /root/packaging/runtime
+
+   # To see the options, arguments, etc
+   ./update_runtime.sh --help
+
+   # to build locally
+   ./update_runtime.sh --local-build --verbose
+
+   # the resulting packages can be found in /var/packaging/results inside the container.
+   # It could be convenient to bind the /var/packaging/results directory to the host:
+   docker run -v /repo/location:/root/packaging -v /var/packages:/var/packaging/results -it cc-osc bash
```

Comment:
This is indeed useful. I'd be tempted to combine this `docker run` with the one above which could then become:

```
$ results=/var/packaging/results
$ sudo mkdir -p "$results"
$ sudo docker run -v $PWD:/root/packaging -v /var/packaging/results:$results -it cc-osc bash
```

And just add a note the the command above to explain that the generated packages will be placed on the host system in the directory defined by `$results`.